src/main/resources/align-confluent-kafka.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Any chance you could add an integration test similar to https://github.com/nebula-plugins/gradle-resolution-rules/blob/main/src/integTest/groovy/nebula/plugin/resolutionrules/AlignSpringKafkaIntegrationSpec.groovy ?